### ISSUE TITLE
MimeTypes Wishlist ✨

### ISSUE THREAD

**Issue by snwh** [snwh/suru-icon-theme#65]
If you can think of any icons for file types you'd like to see in the Suru icon set do comment on this issue below with the following info:

 - the file extension, e.g. `.pdf`
 - the file media type code e.g. `application/pdf` (which you can find in usually in the properties dialog of a file)

And do use Markdown checkboxes in your comment please and thank you :smile: Here's an example for a RAR file.

- [ ] .rar (application/vnd.rar)
```md
- [ ] .rar (application/vnd.rar)
```

![screenshot from 2018-06-21 21-24-46](https://user-images.githubusercontent.com/1217306/41753134-8f7ceccc-7599-11e8-9e02-43e67872ce30.png)

**Comment by yeknava** [comment]
- vue files ([vue](https://vuejs.org/) is a popular js framework)
- dart files ([dart](https://www.dartlang.org/) programming language)
- [ ] .vue (text/plain)
- [ ] .dart (text/plain)

**Comment by snwh** [comment]
@yeknava neither of those are actually registered mimetypes, don't just make it up. :unamused:

**Comment by snwh** [comment]
@yeknava sorry, friend, those aren't registed mimetypes either. Where are you finding this info?
![screenshot from 2018-06-22 12-29-00](https://user-images.githubusercontent.com/1217306/41787928-de22a28a-7617-11e8-9e89-dfdfd025f659.png)
![screenshot from 2018-06-22 12-29-23](https://user-images.githubusercontent.com/1217306/41787949-e9e995b0-7617-11e8-8d7f-145c356f8d33.png)
They are both plain-text, if they had a mimetype they'd appear as such here.

**Comment by yeknava** [comment]
@snwh yes i fixed them just right now, they are both plain-text

**Comment by snwh** [comment]
@yeknava which means they are an unregistered media type and won't have an icon...

**Comment by yeknava** [comment]
@snwh they are languages and frameworks, they would never have registered media types.

**Comment by snwh** [comment]
@yeknava then they will never have an icon. Other languages have registered types. :man_shrugging:

**Comment by ricvelozo** [comment]
- [ ] .scss (`text/x-scss`) - alternative syntax of Sass
- [x] .flatpak (`application/vnd.flatpak`, `application/vnd.flatpak.ref`, `application/vnd.flatpak.repo`)
- [ ] .wasm (`application/wasm`) - not registered yet (WebAssembly/design#1202)
- [ ] .dart (`application/vnd-dart`) - registered, but GNOME Files don't recognises it yet

Another source of mime types is [IANA](https://www.iana.org/assignments/media-types/media-types.xhtml) and your distro:

```console
$ ls /usr/share/mime/application/ | grep flatpak
$ cat /usr/share/mime/application/vnd.flatpak.xml # <mime-type type="application/vnd.flatpak">
```

**Comment by snwh** [comment]
@ricvelozo right, if it's not in `/usr/share/mime/application` then an icon won't show up.

**Comment by ricvelozo** [comment]
@snwh I think so.

**Comment by ricvelozo** [comment]
@snwh I looked for `text/x-scss` but I don't found, so I don't think so, haha. The GNOME Files still recognises it.

**Comment by inferno986return** [comment]
If not already within the iconset I propose an icon for .svg and .svgz files. The mimetype is `image/svg+xml` and the logo is on the W3 website: https://www.w3.org/Graphics/SVG/.

Wikicommons link to image: https://en.wikipedia.org/wiki/Scalable_Vector_Graphics#/media/File:SVG_logo.svg

- [x] `image/svg+xml`

**Comment by snwh** [comment]
@eaglersdeveloper I'm not likely gonna use the windows logo

**Comment by inferno986return** [comment]
A simple Google search shows that cog wheels are a popular motif for .exe icons. It's not original, but I like the metaphor.

**Comment by snwh** [comment]
@inferno986return I've got the exe icon done ;)

**Comment by jnlucian** [comment]
Missing ISO icon

https://mega.nz/#!x0A2QCaL!Bdh1ez_JOjsDhgu2jjrDJKUMoDOtHH9D9a7eKtHohvY

https://mega.nz/#!wxQiGQSI!SDN1CfP_QE-aJ29d1VOyGmatD4e41f4pz5J0xsUuN_s

**Comment by jnlucian** [comment]
icon ISO

![captura de pantalla de 2018-07-20 16-39-22](https://user-images.githubusercontent.com/40614280/43030052-06aac2d4-8c5a-11e8-95bf-e19540253053.png)
![captura de pantalla de 2018-07-20 16-43-21](https://user-images.githubusercontent.com/40614280/43030054-0866cb04-8c5a-11e8-8d9b-9dd58e399b97.png)

**Comment by mymike00** [comment]
- [ ] .gpx (application/gpx+xml)

**Comment by mymike00** [comment]
- [ ] .qml File Qt Markup Language (text/x-qml)
